### DISCUSSION TITLE
Reset the position of Turtlebot 4

### DISCUSSION THREAD

**Discussion by ACdu22330** [turtlebot/turtlebot4#262] [General]
Hello ! 

I'm using the Turtlebot 4 and I have a question about the reset_pose service. 
Is it possible to reset the position of the Turtlebot 4 in a different position from the origin point ?

I've tried with this ros2 command : 

 ros2 service call /reset_pose irobot_create_msgs/srv/ResetPose "pose:
        position:
            x: 1.0
            y: 2.0
            z: 0.0
        orientation:
            x: 0.0
            y: 0.0
            z: 0.0
            w: 0.0
        "

But the robot just reste the position at 0 (as you can see in the 2 screens attached). Does the service reset_pose can change the position of the robot or an other topic or service is capable to do that ? 

Thank you for your answers ! 
![Screenshot from 2023-09-04 15-37-23](https://github.com/turtlebot/turtlebot4/assets/135013797/c0551ecf-6a41-4f04-897e-260da459c224)
![Screenshot from 2023-09-04 15-37-48](https://github.com/turtlebot/turtlebot4/assets/135013797/e85a19ff-e9f7-444c-9dee-6f3cec0955bd)

**Comment by slowrunner** [comment]
The Create3 Reset pose server accepts a pose but does not use it, indeed.

I filed [an issue on this](https://github.com/iRobotEducation/irobot_create_msgs/issues/15).   The response suggests the documentation will be updated to match the existing functionality.
